### PR TITLE
fix: Define array options to yargs

### PR DESCRIPTION
**Pull request by craftzdog**
It allows you to pass array options via CLI arguments.

### CODE REVIEW

**Comment by fcastilloec** [comment]
You can already pass array options with yargs if you specify the option more than once. For example:
```
electron-installer-redhat --src src_dir --dest dest_dir --arch x86_64 --options.categories Audio --options.categories Music
```

That will pass the array `['Audio', 'Music']`

**Comment by craftzdog** [comment]
No, you have to specify at least two values so that the command recognizes the option is an array.
For example, it doesn't work:

```sh
electron-installer-redhat --src src_dir --dest dest_dir --arch x86_64 --options.mimeType x-scheme-handler/inkdrop
```

It recognizes `mimeType` as string, not array, which causes an error.

**Comment by fcastilloec** [comment]
> No, you have to specify at least two values so that the command recognizes the option is an array.
> For example, it doesn't work:
> 
> ```shell
> electron-installer-redhat --src src_dir --dest dest_dir --arch x86_64 --options.mimeType x-scheme-handler/inkdrop
> ```
> 
> It recognizes `mimeType` as string, not array, which causes an error.

Sure, but as a  workaround, you can just do:
```shell
electron-installer-redhat --src src_dir --dest dest_dir --arch x86_64 --options.mimeType x-scheme-handler/inkdrop --options.mimeType ''
```
Only until a new release with this code gets published.

File: src/cli.js [hunk 1]
```diff
@@ -25,6 +25,9 @@ const argv = yargs
     describe: 'JSON file that contains the metadata for your application',
     config: true
   })
+  .options('options.execArguments', { array: true, hidden: true })
+  .options('options.categories', { array: true, hidden: true })
+  .options('options.mimeType', { array: true, hidden: true })
```
Comment (fcastilloec) [inline src/cli.js:30]: can you add one for `options.requires`?

Comment (craftzdog) [reply src/cli.js:31]: Sure

**Comment by craftzdog** [comment]
Added a definition for `options.requires`

**Pull request merged**